lib/rack/honeycomb/middleware.rb

```diff
@@ -67,6 +66,7 @@ def initialize(app, options = {})
           add(
             'meta.package' => package,
             'meta.package_version' => package_version,
+            'type' => EVENT_TYPE,
```

Comment:
If we don't continue to add this here then if you are using just this gem standalone (without the beeline) it will never be set